javascript-modal/main.js

```diff
@@ -1,24 +1,20 @@
 var openModalButton = document.querySelector('.open-modal');
-var openModal = false;
 var overlayElement = document.querySelector('.overlay');
 var modalElement = document.querySelector('.modal');
 var modalButton = document.querySelector('.modal-button');
-var closeModal = false;
 
 openModalButton.addEventListener('click', modalHandle);
 modalButton.addEventListener('click', modalButtonHandle);
 
 function modalHandle(event) {
-  openModal = !openModal;
-  if (openModal) {
+  if (event) {
     overlayElement.className = 'overlay';
     modalElement.className = 'modal';
   }
 }
 
 function modalButtonHandle(event) {
-  closeModal = !closeModal;
-  if (closeModal) {
+  if (event) {
```

Comment:
The event object will always be present in an event callback, so this conditional will always fire. You can just remove the conditionals entirely here 👍